src/main/kotlin/com/semgrep/idea/lsp/SemgrepInstaller.kt

```diff
@@ -72,6 +73,10 @@ object SemgrepInstaller {
         return if (result == "") null else result
     }
 
+    fun isWindows():Boolean{
+        return SystemInfo.isWindows
+    }
```

Comment:
uber-nit: why do we need our own method for this? `SystemInfo.isWindows()` works fine

Reply:
Organizing it this way makes sense to my brain. If I don't put it in this method I will forget this check is here if we ever support windows in the future